graph/pull.go

```diff
@@ -491,7 +491,7 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 
 				io.Copy(tmpFile, utils.ProgressReader(ioutil.NopCloser(tarSumReader), int(l), out, sf, false, utils.TruncateID(img.ID), "Downloading"))
 
-				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum", nil))
+				out.Write(sf.FormatProgress(utils.TruncateID(img.ID), "Verifying Checksum - this is a tech preview, don't rely on it for security yet.", nil))
```

Comment:
well I still think it needs to be said, do you suggest putting it somewhere else so it stays